### ISSUE TITLE
Files not showing and File explorer gone.

### ISSUE THREAD

**Issue by defunky** [jlennox/VSCode-CustomDocumentWell#2]
I've installed version 1.1.0 but it doesn't seem to be working. There are two major bugs. 

- While the sidebar gets created, but it's not populated with currently open files. Can't seem to get anything to display in the new sidebar
- File explorer disappears completely, you can see couple of pixels of it on the right. It seems like the new sidebar pushes the file explorer out of the viewport meaning you can't access any of your files.

I'm using VS code 1.53.2, no other additional extensions installed.

Console does throw exception with error : "UNTRUSTED html usage, default trusted types policy should NEVER be reached"

**Comment by jlennox** [comment]
@defunky I just fixed it up for the latest releases but didn't push a new update because I thought I was the only person using it.

I'll publish an update sometime this week.

Sadly, I don't have a great fix for the untrusted html usage issue, but I'll see if I can at least slipstream it.

Thanks for the report.

**Comment by jlennox** [comment]
For the time being, this should be able to be manually fixed by replacing the text `if(o==='<!DOCTYPE html>` with `if(true || o==='<!DOCTYPE html>` inside of `{vs-code installation dir}\resources\app\out\vs\code\electron-browser\workbench\workbench.js`

and replacing the contents of `{vs-code installation dir}\data\extensions\jlennox0115.custom-document-well-1.1.0\out\customdocumentwell.js` with https://raw.githubusercontent.com/jlennox/VSCode-CustomDocumentWell/master/out/customdocumentwell.js

**Comment by vlascik** [comment]
First of all, thanks for the plugin, vertical tabs is a must for me to be able to use VS Code, and it's a glaring omission on part of MS, so thanks for trying to bring it to life.

However, the last modification helped to show the custom well, but File explorer etc. sidebar contents  are still hidden. 

It seems there is a wrong calculation of `left` style attribute for sidebars somewhere, they are shown under the custom well. E.g. in my case `<div class="split-view-view visible">` element for sidebar content was set to `left: -353px;`

Btw. it might be a good idea to use something like https://github.com/cure53/DOMPurify to sanitize HTML to avoid having to modify Chromium/VSC checks for Trusted Types API.

**Comment by jlennox** [comment]
Thank you for the reply. I haven't yet had a chance to dig into DOMPurify yet.

> but File explorer etc. sidebar contents are still hidden.

Are you using file explorer on the left side? I've only ever tested this with the VS Code side bar being on the right side.

**Comment by vlascik** [comment]
Yes, it's on the left side. Don't know if your script sets the `left` style on that element, but if it is, perhaps it should just set `margin-left: 48px` instead.
